public/app/plugins/datasource/cloud-monitoring/components/AnnotationQueryEditor.tsx

```diff
@@ -40,112 +35,67 @@ const DefaultTarget: State = {
   labels: {},
   variableOptionGroup: {},
   variableOptions: [],
-};
-
-export class AnnotationQueryEditor extends React.Component<Props, State> {
-  state: State = DefaultTarget;
-
-  async UNSAFE_componentWillMount() {
-    // Unfortunately, migrations like this need to go UNSAFE_componentWillMount. As soon as there's
-    // migration hook for this module.ts, we can do the migrations there instead.
-    const { target, datasource } = this.props;
-    if (!target.projectName) {
-      target.projectName = datasource.getDefaultProject();
-    }
-
-    const variableOptionGroup = {
-      label: 'Template Variables',
-      options: datasource.getVariables().map(toOption),
-    };
-
-    const projects = await datasource.getProjects();
-    this.setState({
-      variableOptionGroup,
-      variableOptions: variableOptionGroup.options,
-      ...target,
-      projects,
-    });
-
-    datasource
-      .getLabels(target.metricType, target.projectName, target.refId)
-      .then((labels) => this.setState({ labels }));
-  }
-
-  onMetricTypeChange = ({ valueType, metricKind, type, unit }: MetricDescriptor) => {
-    const { onQueryChange, datasource } = this.props;
-    this.setState(
-      {
-        metricType: type,
-        unit,
-        valueType,
-        metricKind,
-      },
-      () => {
-        onQueryChange(this.state);
-      }
-    );
-    datasource.getLabels(type, this.state.refId, this.state.projectName).then((labels) => this.setState({ labels }));
+  query: '',
+  crossSeriesReducer: 'REDUCE_NONE',
+  perSeriesAligner: AlignmentTypes.ALIGN_NONE,
```

Comment:
For clarity, can we also set alignmentPeriod to `grafana-auto` since that seems to be the default in the backend?